### PR TITLE
Fix and added navigation bar tests, and pinned tab test.

### PR DESCRIPTION
**Pull request by rob-blasetti**

### CODE REVIEW

File: test/pinnedTabTest.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  describe('new tab signal', function () {
+    Brave.beforeAll(this)
+    before(function *() {
+      yield setup(this.app.client)
+    })
+
+    it('creates a new pinned tab when signaled', function *() {
+      yield this.app.client
+        .ipcSend(messages.SHORTCUT_NEW_FRAME, 'http://www.brave.com', { isPinned: true })
+        .waitForExist('.tab[data-frame-key="2"]')
```
Comment (bbondy) [inline test/pinnedTabTest.js]: Please update this to: `.tab.isPinned[data-frame-key="2"]` and make sure the test still works.

File: test/navigationBarTest.js [hunk 2]
```diff
@@ -207,8 +207,9 @@ describe('urlbar', function () {
     })
 
     it('has an empty url with placeholder', function *() {
+      // var placeholder = 'Search or enter an address'
       yield defaultUrlInputValue(this.app.client)
-      yield selectsText(this.app.client, '')
+      // assert.equal(this.getValue(urlInput), placeholder)
```
Comment (bbondy) [inline test/navigationBarTest.js]: nit: Please remove those comments.

**Comment by bbondy** [comment]
This is so great, thanks you a ton for doing this!
Just a couple nits and then I'll merge.

**Comment by rob-blasetti** [comment]
@bbondy they should be fixed now, thanks!

**Comment by bbondy** [comment]
This helps so much, seriously!
Nice work, thanks.

**Pull request merged**